_data/simple-icons.json

```diff
@@ -14146,6 +14146,11 @@
             "hex": "0E3A2F",
             "source": "https://www.skoda-connect.com"
         },
+        {
+            "title": "sktime",
+            "hex": "3FB1C5",
+            "source": "https://www.sktime.net/en/stable/"
```

Comment:
Seeing as the SVG version isn't present on the website itself, let's wait until https://github.com/sktime/sktime/pull/7024 is merged and then grab a permanent GitHub link to the file (https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines).

Reply:
Should be resolved [here](https://github.com/simple-icons/simple-icons/pull/11670/commits/5a28029b2b3834588758e30754c7785ac055670a).

Reply:
As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md?rgh-link-date=2024-08-25T23%3A09%3A36Z#source-guidelines the link must be a permalink.

Reply:
Sorry fixed [here](https://github.com/simple-icons/simple-icons/pull/11670/commits/abbd55f222b5c6e8c6bf837b0210dbf117df8092).

Reply:
Sweet! I will take a look later.